vpue_undergrad_subtheme.libraries.yml

```diff
@@ -100,3 +100,9 @@ allpages:
     theme:
       dist/css/theme.css: {}
       dist/css/print.css: { media: print }
+
+fonts:
+  css:
+    theme:
+      'fonts.googleapis.com/css2?family=Lora:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&display=swap': { type: external, minified: true }
```

Comment:
@jdwjdwjdw can you try this line starting with // like I did here:
https://github.com/SU-SWS/saa_victoria_subtheme/commit/170e3f52b59cc78a8a5adfc0f3f15676c0483988
Or try with http:// for testing to get it to work initially. The // format will allow the site to load it with either http or https depending what the site requests or uses by default, I believe.
I would try http:// version on that line 107 as a first debugging step and see if it works.

Reply:
Thanks @katrialesser! Looks like that did the trick 😄